packages/bot-web-ui/src/stores/flyout-help-store.js

```diff
@@ -96,13 +95,32 @@ export default class FlyoutHelpStore {
             }
         };
 
-        const block_type = await getNextBlock(xml_list_group,current_block_index,should_go_next);
+        const block_type = await getNextBlock(this.xml_list_group,current_block_index,should_go_next);
         if (block_type) {
-            const target_blocks = xml_list_group[block_type];
+            const target_blocks = this.xml_list_group[block_type];
             this.setHelpContent(target_blocks[0]);
         }
     }
 
+    @action.bound
+    initializeFlyoutHelp(block_node) {
+        // eslint-disable-next-line no-underscore-dangle
+        const toolbox = Blockly.derivWorkspace.toolbox_;
+        const selected_category = toolbox.getSelectedItem();
+        this.xml_list = toolbox.getCategoryContents(selected_category);
+        this.xml_list_group = this.groupBy(this.xml_list, true);
+
+        this.setHelpContent(block_node);
+    }
+
+    @action.bound
+    shoudDisableSequence() {
```

Comment:
Could we rename this to something like `updateSequenceButtons`? The current name suggests a boolean will be returned.